js/packages/cli/src/helpers/constants.ts

```diff
@@ -100,7 +100,7 @@ export const CONFIG_LINE_SIZE = 4 + 32 + 4 + 200;
 
 export const CACHE_PATH = './.cache';
 
-export const DEFAULT_TIMEOUT = 15000;
+export const DEFAULT_TIMEOUT = 150000;
```

Comment:
That is not good idea... just increase it to 60s or something I think that should be enough.